mix.exs

```diff
@@ -20,7 +20,7 @@ defmodule PhoenixSlime.Mixfile do
   end
 
   def deps do
-    [{:phoenix, "~> 1.2"},
+    [{:phoenix, "~> 1.3-rc"},
```

Comment:
I think we can stick with`~> 1.2` and use `allow_pre: false`, can't we?

https://hexdocs.pm/elixir/Version.html